src/main/java/uk/gov/justice/hmpps/prison/api/resource/ScheduleResource.java

```diff
@@ -117,33 +119,33 @@ public List<PrisonerSchedule> getActivitiesAtAllLocationsByDateRange(@PathVariab
     @Operation(summary = "Get all Prisoner activities for given date.", description = "Get count of suspended prisoner activities for given date range")
     @GetMapping("/{agencyId}/count-activities")
     public PrisonerActivitiesCount getCountActivitiesByDateRange(@PathVariable("agencyId") @Parameter(description = "The prison.", required = true, example = "MDI") final String agencyId,
-                                                                 @RequestParam(value = "fromDate", required = false) @org.springframework.format.annotation.DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "From date of whereabouts list, default today") final LocalDate fromDate,
-                                                                 @RequestParam(value = "toDate", required = false) @org.springframework.format.annotation.DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @Parameter(description = "To Date of whereabouts list, default from date") final LocalDate toDate,
+                                                                 @RequestParam(value = "fromDate", required = true) @DateTimeFormat(iso = ISO.DATE) @Parameter(description = "From date of whereabouts list") @NotNull final LocalDate fromDate,
```

Comment:
whereabouts always passed the from and to dates, so no reason for these not to be mandatory